### ISSUE TITLE
Odd layout in Hausa translation

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecule-shapes#250]
**Test device**
Dell
**Operating System**
Win 11
**Browser**
Chrome
**Problem description**
Found during https://github.com/phetsims/qa/issues/913, but it is in published. I also saw similar behavior in one or two other translations, but it was most pronounced in the `ha` sim.
The box that contains the molecule shape names becomes ridiculously stretched out in the `ha` translation of the sim.

**Visuals**
<img width="641" alt="halocale" src="https://user-images.githubusercontent.com/41024075/223839741-e3865357-fa26-481c-9c80-17ae4e9c4f6d.png">

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Molecule Shapes‬
URL: https://phet-dev.colorado.edu/html/molecule-shapes/1.6.0-rc.3/phet/molecule-shapes_all_phet.html
Version: 1.6.0-rc.3 2023-03-08 03:19:35 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36
Language: en-US
Window: 1280x649
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jonathanolson** [comment]
There is a very long string for one of the Molecule Geometries. It's a bit unfortunate, but I think this is as expected/desired, since we want the string to be readable.

**Comment by jonathanolson** [comment]
<img width="884" alt="image" src="https://user-images.githubusercontent.com/480878/223876951-ff291bb7-f43e-4f3b-a8ff-ef49ff37790f.png">

**Comment by jonathanolson** [comment]
@kathy-phet or @arouinfar, thoughts?

**Comment by kathy-phet** [comment]
Agreed. Best option would be to ask the translator to refine/shorten.

**Comment by arouinfar** [comment]
I ran the Hausa geometry strings through Google Translate, and they all seem appropriate, if wordy. It's possible the locale doesn't have a succinct term for things like "Trigonal Planar" or "Tetrahedral". The layout can accommodate long strings, so I'm going to close.
